app/Login.tsx

```diff
@@ -46,13 +45,18 @@ export function Login(props: { onLoginSuccess: () => void }): ReactElement {
             display: "flex",
             flexDirection: "column",
             flexGrow: 4,
-            color: "#FFFFFF",
             fontSize: "12px",
           }}
         >
           <Form.Label>Username</Form.Label>
+          <div style={{ height: "10px" }}></div>
```

Comment:
What's this div for? If things are spaced away from each other let's use `margin`

Reply:
replaced this div with marginBottom because a regular margin indented the form label